<ui version="4.0" >
 <class>JobWindow</class>
 <widget class="QMainWindow" name="JobWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>JobTimer</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTableView" name="jobTable" >
      <property name="acceptDrops" >
       <bool>true</bool>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="dragEnabled" >
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode" >
       <bool>false</bool>
      </property>
      <property name="dragDropMode" >
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="cornerButtonEnabled" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionStart" />
   <addaction name="actionStop" />
   <addaction name="actionRemove" />
   <addaction name="actionDone" />
  </widget>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionStart" >
   <property name="text" >
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionRemove" >
   <property name="text" >
    <string>Remove</string>
   </property>
  </action>
  <action name="actionDone" >
   <property name="text" >
    <string>Done</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>JobWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>400</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
